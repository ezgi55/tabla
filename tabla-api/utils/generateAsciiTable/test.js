const generateAsciiTable = require(".");

it("should generate an ASCII table", () => {
  const output = generateAsciiTable({
    title: "Hello Table",
    columnHeaders: ["Col 1", "Column 2", "Some Other Column"],
    rows: [
      ["162374", "ahskhdfkj", "asdasdf"],
      ["askjdhf", "asdfasdfsa", "asdfasdf"],
      ["what a beatiful table", "cell", "celll"]
    ]
  });

  expect(output).toMatchSnapshot();
});
