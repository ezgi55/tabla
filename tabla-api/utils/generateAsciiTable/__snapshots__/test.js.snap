// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate an ASCII table 1`] = `
".--------------------------------------------------------.
|                      Hello Table                       |
|--------------------------------------------------------|
|         Col 1         |  Column 2  | Some Other Column |
|-----------------------|------------|-------------------|
| 162374                | ahskhdfkj  | asdasdf           |
| askjdhf               | asdfasdfsa | asdfasdf          |
| what a beatiful table | cell       | celll             |
'--------------------------------------------------------'"
`;
 